services/src/test/groovy/org/jfrog/artifactory/client/BaseRepositoryTests.groovy

```diff
@@ -84,6 +91,32 @@ abstract class BaseRepositoryTests extends ArtifactoryTestsBase {
                     .customProperties(customProperties)
                     .build()
         }
+        if (prepareFederatedRepo) {
+            List<FederatedMember> members = new ArrayList<FederatedMember>()
+            if(federationUrl!=null && StringUtils.isNoneBlank(federationUrl))
+            {
```

Comment:
Curly bracket should be moved one line above, according to oir coding conventions. Also mind the white-space.

Reply:
fixed